resources/lang/en/patrols/forest/border/any.json

```
@@ -307,7 +350,10 @@
                         "cats_from": ["clan"],
                         "mutual": false,
                         "values": ["trust", "respect"],
-                        "amount": 15
+                        "amount": 10,
+                        "log": {
+                            "cats_from": "from_cat saw how to_cat put {PRONOUN/to_cat/poss} life on the line in battle against some rogues."
```

```suggestion
                            "cats_from": "from_cat heard how to_cat put {PRONOUN/to_cat/poss} life on the line in battle against some rogues."
```
since this is a clan reaction, not patrol